contents/docs/advanced/proxy/cloudfront.mdx

```diff
@@ -38,21 +38,21 @@ export const diagram = "https://res.cloudinary.com/dmukukwp6/image/upload/postho
 
 6. Back on the “Create distribution” page, select your “origin-cors” cache policy.
 7. Under “Origin request policy,” choose “CORS-CustomOrigin.”
-8. Under “Response headers policy,” choose “CORS-with-preflight-and-SecurityHeadersPolicy.”
-9. Do not enable AWS Web Application Firewall (WAF).
-10. Click “Create distribution” at bottom of the page.
-11. Edit the Cloudfront Distribution and add the assets origin `us-assets.i.posthog.com` or `eu-assets.i.posthog.com`.
-12. Add another behavior the path will be `/static/*`.
-13. Under "Origin,", choose the one created above.
-14. Under “cache policy,” choose “origin-cors”
-15. Under “Origin request policy,” click “Create origin request policy.” On the “Create origin request policy” page:
+8. Do not enable AWS Web Application Firewall (WAF).
+9. Click “Create distribution” at bottom of the page.
```

Comment:
```suggestion
9. Click **Create distribution** at bottom of the page.
```


We tend to [bold buttons and tab names for easier scanning](https://posthog.com/handbook/content/docs-style-guide#bold-button-and-tab-names)